packages/loader/container-loader/src/loader.ts

```diff
@@ -408,17 +420,21 @@ export class Loader extends EventEmitter implements ILoader {
         return { container, parsed };
     }
 
+    private mergeHeaderOptions(headers: IRequestHeader): IRequestHeader {
+        const loaderDefaults = {
+            [LoaderHeader.cache]: this.services.options.cache,
+            [LoaderHeader.pause]: this.services.options.pause,
+            [LoaderHeader.reconnect]: this.services.options.reconnect,
+        };
+        return { ...loaderDefaults, ...headers };
```

Comment:
I think there should be only one way of specifying these things. I.e. LoaderHeader.pause should be deleted and host should be forced to use options and have no way to override it for an instance of the container. As otherwise loader options are meaningless - they are supposed to represent constant things about all containers in loader.